### PR TITLE
Add Espresso configuration

### PR DESCRIPTION
**Pull request by valentyn88**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

Add Espresso configuration to the docs. 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by vrunoa — APPROVED** [review]

File: docs/testrunner-toolkit/configuration/espresso.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+__Example__:
+```yaml
+  - name: "Android GoogleApi Emulator"
+```
+
+### `orientation`
+
+__Description__: Screen orientation. 
+
+__Type__: *string*
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/espresso.md:83]: Is this a string or an enum?

File: docs/testrunner-toolkit.md [hunk 2]
```diff
@@ -33,7 +34,7 @@ JavaScript framework of your choice.
 
 ## Supported Frameworks and Browsers
 
-In the current beta, the toolkit supports the following frameworks: [Cypress](https://github.com/cypress-io/cypress), [Playwright](https://github.com/microsoft/playwright), and [TestCafe](https://github.com/DevExpress/testcafe). The specific framework you use for testing depends on the types of tests you
+In the current beta, the toolkit supports the following frameworks: [Cypress](https://github.com/cypress-io/cypress), [Playwright](https://github.com/microsoft/playwright), [TestCafe](https://github.com/DevExpress/testcafe), and [Espresso](https://developer.android.com/training/testing/espresso). The specific framework you use for testing depends on the types of tests you
 need to run, and the environment where you run the tests.
 
 Furthermore, the framework and browser version support depends on the Sauce Labs docker images. The table below indicates framework and browser support based on the requisite docker image tag.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit.md:40]: Do we need to add an Espresso tab to each of the tables to show the supported OS/browser combinations?

**Review by sweeneyskirt-sl — COMMENTED** [review]
Hi @valentyn88  -- I just have a couple questions in this review. I don't mind merging your changes, but please keep in mind that some of the changes on the Intro page (testrunner-toolkit.md) will be overridden by the rewrite I have done. I will, however, make sure that the tenet of your changes is preserved.

File: docs/testrunner-toolkit/configuration/espresso.md (hunk 1 above)
Comment (vrunoa) [reply docs/testrunner-toolkit/configuration/espresso.md:83]: orientation is either `portrait` or `landscape`, it can be treated as a enum

**Review by vrunoa — APPROVED** [review]

File: docs/testrunner-toolkit.md (hunk 2 above)
Comment (valentyn88) [reply docs/testrunner-toolkit.md:40]: The supported platform is only Android and there are no browser combinations, only supported Android versions.

File: docs/testrunner-toolkit/configuration/espresso.md (hunk 1 above)
Comment (valentyn88) [reply docs/testrunner-toolkit/configuration/espresso.md:83]: I changed this field to the `enum` and also added possible values.

**Comment by valentyn88** [comment]
> Hi @valentyn88 -- I just have a couple questions in this review. I don't mind merging your changes, but please keep in mind that some of the changes on the Intro page (testrunner-toolkit.md) will be overridden by the rewrite I have done. I will, however, make sure that the tenet of your changes is preserved.

Hey, @sweeneyskirt-sl ! Thanks for your review! Just keep in mind changes that will be added by this PR.

**Review by sweeneyskirt-sl — APPROVED** [review]

**Pull request merged**